src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,55 @@
+.stopwatch {
+  position: absolute;
+  width: 500px;
+  height: 500px;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border: 10px solid rgb(127, 255, 212);
+  background-color: #fff;
+  border-radius: 50%;
+  margin: auto;
+  box-shadow: 20px 20px 10px 15px rgba(0, 0, 255, 0.2);
+}
+
+.stopwatch__center {
```

Comment:
- `center` itself should be placed in front of all the other elements (hint – use `z-index` property)
- let's use nesting

Reply:
i have used z-index so now the .center is covering the watch hands.